Detectors/MUON/MID/Base/include/MIDBase/Mapping.h

```diff
@@ -86,10 +86,9 @@ class Mapping
   void buildDETypeMedium(int rpcType, std::array<int, 7> boards, bool largeNonBend);
   void buildDETypeCut(int rpcType, std::array<int, 7> boards, bool isBelowBeamPipe);
   void buildDETypeShort(int rpcType, std::array<int, 7> boards);
-  int getChamber(int deId) const;
-  int getRPCType(int deId) const;
+  /// Gets the RPC type of detection element deId
+  inline int getRPCType(int deId) const { return deId % 9; }
```

Comment:
note that inline is only needed outside of the class body declaration. But it does not hurt (just mentioned it for the next time).

Reply:
HI @sawenzel ,
ok, thanks a lot for the suggestion: I'll take it into account for the next PR..
Cheers,
Diego